python/lsst/sims/utils/AstrometryUtils.py

```diff
@@ -119,7 +120,8 @@ def refractionCoefficients(wavelength=0.5, site=None):
     precision = 1.e-10
 
     if site is None:
-        raise RuntimeError("Cannot call refractionCoefficients; no site information")
+        raise RuntimeError(
+            "Cannot call refractionCoefficients; no site information")
```

Comment:
I think this means that `autopep8` was run without specifying a maximum line length. For DM it's 110 characters but maybe Sims uses the Python default of 80.